### ISSUE TITLE
Typography Sass File Location

### ISSUE THREAD

**Issue by buschschwick** [factor1/factor1_blankwp#13]
While starting up a new project I questioned if `components` is the correct location for the `_typography.scss` file.

Would it make more sense in `settings` or `global`?

@factor1/staff

**Comment by buschschwick** [comment]
Yeah, that's what I was leaning towards as well. 

Typical `_variables.scss` looks sorta like this in the early stages:

```
// Variables

// Colors
$ltblue:		#00a9e0;
$drkblue:		#2f2a95;
$yellow:		#fed141;
$gold:			#ffad00;


// Fonts
$proxima:		'proxima-nova', Helvetica, sans-serif;
```

and then `_typography.scss` just references the variables.

**Comment by buschschwick** [comment]
Added this to be released in 2.1.0

**Comment by buschschwick** [comment]
Added to `2.1.0` branch with commit https://github.com/factor1/factor1_blankwp/commit/f9a4075330520c23834c2dd8245ee1cbe407b345
